src/libs/ajax/GoogleStorage.ts

```diff
@@ -89,17 +89,17 @@ const fetchBuckets = _.flow(
   withUrlPrefix('https://storage.googleapis.com/')
 )(fetchOk);
 
-/**
- * Only use this if the user has write access to the workspace to avoid proliferation of service accounts in projects containing public workspaces.
- * If we want to fetch a SA token for read access, we must use a "default" SA instead (api/google/user/petServiceAccount/token).
- */
 const getServiceAccountToken: (googleProject: string, token: string) => Promise<string> = Utils.memoizeAsync(
   async (googleProject, token) => {
+    const workspace = workspaceStore.get();
```

Comment:
Is this the proper way to get the workspace? I did not want to plumb it down through the parameters of all the invocations.